packages/server/db.ts

```diff
@@ -1,24 +1,39 @@
 import { Client } from 'pg';
+import { Sequelize, SequelizeOptions } from 'sequelize-typescript';
 
 const { POSTGRES_USER, POSTGRES_PASSWORD, POSTGRES_DB, POSTGRES_PORT } = process.env;
 
-export const createClientAndConnect = async (): Promise<Client | null> => {
+const sequelizeOptions: SequelizeOptions = {
+  host: 'localhost',
+  port: Number(POSTGRES_PORT),
+  username: POSTGRES_USER,
+  password: POSTGRES_PASSWORD,
+  database: POSTGRES_DB,
+  dialect: 'postgres',
+  models: [__dirname + '\\src\\Models'],
```

Comment:
Так работать будет только на винде  
Чтобы работало на всех ОС (в докере в том числе), надо заюзать `path.join`  

```suggestion
  import path from 'node:path';
  // ...
  models: [path.join(__dirname, 'src', 'Models')],
```